src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Crypto.cs

```diff
@@ -98,7 +98,7 @@ internal static partial class Crypto
         internal static string GetX509RootStorePath()
         {
             IntPtr ptr = GetX509RootStorePath_private();
-            return ptr != null ?
+            return ptr != IntPtr.Zero ?
```

Comment:
> We shouldn't need these checks at all: PtrToStringAnsi is both implemented and documented to return null when passed IntPtr.Zero.

Valid point, pushed out a replacement commit that removes the conditional expression altogether.